src/services/business.tsx

```diff
@@ -0,0 +1,49 @@
+import { Api_Methods } from "../utils/common/enum"; // Importing Api_Methods
+
+import { apiService } from "./apiService";
+
+// USE these example api as refrence for creating different resource api helper service
+// eslint-disable-next-line prettier/prettier
+
+export const apiHelperService = {
+  getAllBusiness: async (params = {}) => {
+    return apiService({
+      method: Api_Methods.GET,
+      endpoint: "/business/all",
+      params,
+    });
+  },
+
+  getAllBusinessPersonalInfo: async (itemId: string) => {
+    return apiService({
+      method: Api_Methods.GET,
+      endpoint: `/business/${itemId}`,
+    });
+  },
+
+  // more api example
+
+  updateItem: async (itemId: string, body: Record<string, any>) => {
```

Comment:
What is the /items endpoint, if it is not a part of business then move it away from this file